src/fparser/two/utils.py

```diff
@@ -380,9 +411,45 @@ def __new__(cls, string, parent_cls=None):
             errmsg = u"{0}: '{1}'".format(cls.__name__, string)
         raise NoMatchError(errmsg)
 
+    def get_root(self):
+        '''
+        Returns the root of the parse tree to which this node belongs.
+
+        :returns: the node at the root of the tree.
+        :rtype: :py:class:`fparser.two.utils.Base`
+
+        '''
+        parent = self
```

Comment:
I would change the `parent` variable to `current`

Reply:
Good idea.